src/phpFastCache/Drivers/Zenddisk/Driver.php

```diff
@@ -0,0 +1,141 @@
+use phpFastCache\Exceptions\phpFastCacheDriverCheckException;
+use phpFastCache\Exceptions\phpFastCacheDriverException;
+use Psr\Cache\CacheItemInterface;
+
+/**
+ * Class Driver (zend disk cache)
+ * Requires Zend Data Cache Functions from ZendServer
+ * @package phpFastCache\Drivers
+ */
+class Driver extends DriverAbstract
+{
+    /**
+     * Driver constructor.
+     * @param array $config
+     * @throws phpFastCacheDriverException
+     */
+    public function __construct(array $config = [])
+    {
+        $this->setup($config);
+
+        if (!$this->driverCheck()) {
+            throw new phpFastCacheDriverCheckException(sprintf(self::DRIVER_CHECK_FAILURE, $this->getDriverName()));
+        }
+    }
+
+    /**
+     * @return bool
+     */
+    public function driverCheck()
+    {
+        if (extension_loaded('Zend Data Cache') && function_exists('zend_disk_cache_store')) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * @param \Psr\Cache\CacheItemInterface $item
+     * @return mixed
+     * @throws \InvalidArgumentException
+     */
+    protected function driverWrite(CacheItemInterface $item)
+    {
+        /**
+         * Check for Cross-Driver type confusion
+         */
+        if ($item instanceof Item) {
+            $ttl = $item->getExpirationDate()->getTimestamp() - time();
+
+            return zend_disk_cache_store($item->getKey(), $this->driverPreWrap($item), ($ttl > 0 ? $ttl : 0));
+        } else {
+            throw new \InvalidArgumentException('Cross-Driver type confusion detected');
+        }
+    }
+
+    /**
+     * @param \Psr\Cache\CacheItemInterface $item
+     * @return mixed
+     */
+    protected function driverRead(CacheItemInterface $item)
+    {
+        $data = zend_disk_cache_fetch($item->getKey());
+        if ($data === false) {
+            return null;
+        }
+
+        return $data;
+    }
+
+    /**
+     * @param \Psr\Cache\CacheItemInterface $item
+     * @return bool
+     * @throws \InvalidArgumentException
+     */
+    protected function driverDelete(CacheItemInterface $item)
+    {
+        /**
+         * Check for Cross-Driver type confusion
+         */
+        if ($item instanceof Item) {
+            return zend_disk_cache_delete($item->getKey());
+        } else {
+            throw new \InvalidArgumentException('Cross-Driver type confusion detected');
+        }
+    }
+
+    /**
+     * @return bool
+     */
+    protected function driverClear()
+    {
+        return @zend_disk_cache_clear();
+    }
+
+    /**
+     * @return bool
+     */
+    protected function driverConnect()
+    {
+        return true;
+    }
+
+    /********************
+     *
+     * PSR-6 Extended Methods
+     *
+     *******************/
+
+    /**
+     * @return driverStatistic
+     */
+    public function getStats()
+    {
+        $stat = new driverStatistic();
+        $stat->setInfo('[ZendDisk] A void info string')
```

Comment:
Exacxtly, Zenddisk has no statistic, so can you specify it in setInfo() please ?